config/config.sample.php

```diff
@@ -2182,6 +2182,16 @@
 
 'login_form_autocomplete' => true,
 
+/**
+ * If your user is using an outdated browser, a warning will be shown
+ * to offer some guidance to upgrade and ensure a proper Nextcloud experience.
+ * They can still bypass it after they have read the warning.
+ *
+ * Simply set this property to "false", if you want to turn this feature off.
```

Comment:
If I understand correctly.
```suggestion
 * Simply set this property to "true", if you want to turn this feature off.
```

Reply:
:sob: I just rebased everything!

Nice catch..... 🥲

Reply:
Fixed